src/sentry/api/endpoints/team_avatar.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import
+
+from sentry.api.bases.avatar import AvatarMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.models import TeamAvatar
+
+
+class TeamAvatarEndpoint(AvatarMixin, TeamEndpoint):
+    object_type = 'team'
+    model = TeamAvatar
+
+    def get(self, request, team):
+        return super(TeamAvatarEndpoint, self).get(request, team)
```

Comment:
i did this because in `Endpoint`, `team` is passed as a kwarg and without it, this happens:
```
Traceback (most recent call last):
  File "/Users/jess/code/getsentry/sentry/src/sentry/api/base.py", line 86, in handle_exception
    return super(Endpoint, self).handle_exception(exc)
  File "/Users/jess/code/getsentry/sentry/src/sentry/api/base.py", line 163, in dispatch
    response = handler(request, *args, **kwargs)
TypeError: put() got an unexpected keyword argument 'team'
```
Not sure if there's a better way to fix?